packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -79,6 +88,15 @@ const getMenuVisibilityOfMainmenu = (menu: TNavbarMenu) =>
   menu.menuVisibility ? [menu.menuVisibility] : [];
 
 const ApplicationMenu = (props: ApplicationMenuProps) => {
+  const [topPosition, setTopPosition] = useState(0);
+  const elementRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (elementRef.current != null) {
+      setTopPosition(elementRef.current.offsetTop);
+    }
+  }, [elementRef.current]);
```

Comment:
I don't think this ref ever changes so I'm not sure about adding that to the deps array.

Reply:
Yes, good point. It's always the same so no need for re-runs. Thanks! [c02833c](https://github.com/commercetools/merchant-center-application-kit/pull/3169/commits/c02833c4529801c0282dd2366d22e5dd99f2840e)